cli/src/commands/developer/decrypt.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+use super::CurrentNetwork;
+
+use snarkvm::{
+    console::program::Ciphertext,
+    prelude::{Record, ViewKey},
+};
+
+use anyhow::{bail, Result};
+use clap::Parser;
+use std::str::FromStr;
+
+/// Decrypts a record ciphertext.
+#[derive(Debug, Parser)]
+pub struct Decrypt {
+    /// The record ciphertext to decrypt.
+    #[clap(short = 'c', long, help = "The record ciphertext to decrypt.")]
+    pub ciphertext: String,
+    /// The view key used to decrypt the record ciphertext.
+    #[clap(short = 'v', long, help = "The view key used to decrypt the ciphertext.")]
+    pub view_key: String,
```

Comment:
Clap automagically creates the short from the first letter of the variable, and the comment acts as the help, so doesn't need to be explicitly specified if you don't want.

```suggestion
    /// The record ciphertext to decrypt.
    #[clap(short, long)]
    pub ciphertext: String,
    /// The view key used to decrypt the record ciphertext.
    #[clap(short, long)]
    pub view_key: String,
```